application/src/main/java/be/jpendel/application/SignupUC.java

```diff
@@ -11,6 +11,8 @@ public SignupUC(MemberRepository memberRepository) {
         this.memberRepository = memberRepository;
     }
 
+    //TODO brrrr 'new Member' ipv een builder
+    // Volgens mijn begrip mag de applicatie laag niet een Member rechtreeks aanmaken ..waar is CreateMemberCommand ?
```

Comment:
De UC orchestreert het wat. Hij moet inderdaad niet de hoe bevatten. Maar zal typisch bestaan uit:  haal wat gegevens op, laat commando uitvoeren, persisteer

Dat commando kan dan een new zijn. Maar beter is natuurlijk zoals in kris zijn vb.